scripts/updateMainReference.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021, RTE (http://www.rte-france.com)
+# See AUTHORS.txt
+# All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+
+import os
+import iidmDiff
+import shutil
+
+# Execution requires to have iidmDiff module from dynawo library scripts
```

Comment:
Please add a usage line in the comment, and how to add the path into pythonpath so that everyone can use this without asking for help

Reply:
OK